cores/arduino/WSerial.h

```diff
@@ -4,6 +4,10 @@
 #include "variant.h"
 #include "HardwareSerial.h"
 
+#if defined(USE_TINYUSB)
+#include "Adafruit_USBD_CDC.h"
```

Comment:
Please let me know if any problems arise from adding `#include <Adafruit_TinyUSB.h>` here.
I believe that it is more convenient by including this file.